src/DatabaseLibrary/query.py

```diff
@@ -25,11 +25,10 @@ class Query:
     """
 
     def query(
-        self, selectStatement: str, sansTran: bool = False, returnAsDict: bool = False, alias: Optional[str] = None
+        self, selectStatement: str, sansTran: bool = False, returnAsDict: bool = False, alias: Optional[str] = None, parameters: Optional[List] = None
     ):
         """
-        Uses the input ``selectStatement`` to query for the values that will be returned as a list of tuples. Set
-        optional input ``sansTran`` to True to run command without an explicit transaction commit or rollback.
```

Comment:
I removed all Set Optional.. sentences because they are already in the doc string - at the bottom (so we had it twice for every keyword).